### ISSUE TITLE
Incorrect Spanish Numbers 

### ISSUE THREAD

**Issue by Covidgraphcom** [CSSEGISandData/COVID-19#2284]
**Hopkins:**
_Cases:_
18/04: 191726
17/04: 190839 INCORRECT
16/04: 184948 INCORRECT
15/04: 177644 INCORRECT
14/04: 172541

_Deaths:_
18/04: 20048
17/04: 20002 INCORRECT
16/04: 19315 INCORRECT
15/04: 18708 INCORRECT
14/04: 18056

**[Official Spanish Government](https://www.mscbs.gob.es/):**
_Cases:_
[18/04:](https://www.mscbs.gob.es/profesionales/saludPublica/ccayes/alertasActual/nCov-China/documentos/Actualizacion_79_COVID-19.pdf)  191726 Numbers match
[17/04:](https://www.mscbs.gob.es/profesionales/saludPublica/ccayes/alertasActual/nCov-China/documentos/Actualizacion_78_COVID-19.pdf) 188068
[16/04:](https://www.mscbs.gob.es/profesionales/saludPublica/ccayes/alertasActual/nCov-China/documentos/Actualizacion_77_COVID-19.pdf) 182816
[15/04:](https://www.mscbs.gob.es/profesionales/saludPublica/ccayes/alertasActual/nCov-China/documentos/Actualizacion_76_COVID-19.pdf)  177633 
[14/04:](https://www.mscbs.gob.es/profesionales/saludPublica/ccayes/alertasActual/nCov-China/documentos/Actualizacion_75_COVID-19.pdf): 172541 Numbers match

_Deaths:_
[18/04:](https://www.mscbs.gob.es/profesionales/saludPublica/ccayes/alertasActual/nCov-China/documentos/Actualizacion_79_COVID-19.pdf) 20043 Numbers match
[17/04:](https://www.mscbs.gob.es/profesionales/saludPublica/ccayes/alertasActual/nCov-China/documentos/Actualizacion_78_COVID-19.pdf)  19478
[16/04:](https://www.mscbs.gob.es/profesionales/saludPublica/ccayes/alertasActual/nCov-China/documentos/Actualizacion_77_COVID-19.pdf)  19130
[15/04:](https://www.mscbs.gob.es/profesionales/saludPublica/ccayes/alertasActual/nCov-China/documentos/Actualizacion_76_COVID-19.pdf) 18579
[14/04:](https://www.mscbs.gob.es/profesionales/saludPublica/ccayes/alertasActual/nCov-China/documentos/Actualizacion_75_COVID-19.pdf)  18056 Numbers match

This is important because the websites using Johns Hopkin data as a source show a steep decline in daily new cases and daily new deaths for spain, which is greatly misleading and far from reality.

**Comment by gtancredi** [comment]
It should be 20453.
In the JHU-CSSE webpage, you have the correct figure.

**Comment by Covidgraphcom** [comment]
> It should be 20453.
> In the JHU-CSSE webpage, you have the correct figure.

Sorry, I should have precised that I was referring to the time serie (time_series_covid19_confirmed_global.csv and time_series_covid19_deaths_global.csv).

**Comment by alfkoehn** [comment]
Indeed, the official numbers can be found as daily summary reports at the following locations: 

- 2020-04-17: [1]
- 2020-04-16: [2]
- 2020-04-15: [3]

Maybe this is because the data from the official Spanish source were taken at roughly 21:00 (as written in their report), where as JHU takes them at around midnight. 

Thanks for pointing this out!

[1] https://www.mscbs.gob.es/profesionales/saludPublica/ccayes/alertasActual/nCov-China/documentos/Actualizacion_78_COVID-19.pdf
[2] https://www.mscbs.gob.es/profesionales/saludPublica/ccayes/alertasActual/nCov-China/documentos/Actualizacion_77_COVID-19.pdf
[3] https://www.mscbs.gob.es/profesionales/saludPublica/ccayes/alertasActual/nCov-China/documentos/Actualizacion_76_COVID-19.pdf
